test/parallel/test-http2-client-rststream-before-connect.js

```diff
@@ -28,6 +28,18 @@ server.listen(0, common.mustCall(() => {
   );
   assert.strictEqual(req.closed, false);
 
+  [true, 1, {}, [], 'test'].forEach((notFunction) => {
```

Comment:
Sorry, I should have said this before: can you add `null` here.

Reply:
Done!